packages/cashier/src/containers/routes/binary-routes.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { Switch } from 'react-router-dom';
-import { Localize, PlatformContext } from '@deriv/shared';
+import { Localize } from '@deriv/shared';
 import getRoutesConfig from 'Constants/routes-config';
 import RouteWithSubRoutes from './route-with-sub-routes.jsx';
 
 const BinaryRoutes = props => {
-    const { is_appstore } = React.useContext(PlatformContext);
```

Comment:
we can remove is_appstore because as per bala.krishna discussion:
```
[bala.krishna]
[5:24]
it's safe to remove it
[5:24]
we are handling appstore in a separate git branch
[5:24]
you can go ahead and remove is_appstore flag
```